src/ext/java/org/opentripplanner/ext/siri/SiriAlertsUpdateHandler.java

```diff
@@ -69,10 +65,13 @@
 
     private final String feedId;
 
+    private final Graph graph;
+
     private final Set<TransitAlert> alerts = new HashSet<>();
 
-    public SiriAlertsUpdateHandler(String feedId) {
+    public SiriAlertsUpdateHandler(String feedId, Graph graph) {
```

Comment:
I think you should pass in the `graphTimezone` here, not the graph. 

If all you need is the "salt", why do you pass the "dinner table"?
(Sorry for my bad humor ;-)